src/components/visualizations/chart/legend/LegendList.tsx

```diff
@@ -23,10 +30,17 @@ export default class LegendList extends React.PureComponent<any, any> {
     public render() {
```

Comment:
Pls could you define prop interface for LegendList, understand that series have no types so define this as any. Than you can define a component as React.PureComponent<ILegendListProps> and in the end, you remove any on line  31 const { series, chartType, onItemClick, width } = this.props;